### ISSUE TITLE
No settings for TXDelay or TXTail via KISS

### ISSUE THREAD

**Issue by na7q** [markqvist/MicroAPRS#14]
Currently there is no way set the transmit delay or tail in the KISS version. Website states there is, but no documentation can be found anywhere on this or how it can be done for applications like Xastir or APRSDroid.

**Comment by markqvist** [comment]
It uses the default KISS commands for this. In Xastir you just configure it in the settings, and Xastir sends the commands on modem initialization setting the parameters. I don't know if APRSDroid supports setting these over KISS. Otherwise you can just change the parameters in the source code and compile a version with those values baked in.

**Comment by na7q** [comment]
Once again, there's no documentation on that. And NO, Xastir configuration settings DO NOT get applied. Regardless of the setting, it's still short and does not change. Care to share what to change in the source? I think that'll be my only option.

**Comment by markqvist** [comment]
What hardware are you running on? Changing the settings work fine here.

**Comment by markqvist** [comment]
I just took a look at the Xastir source code, and I can see that the HUPCL flag is hardcoded when opening a serial code, which will toggle the DTR line, thus rebooting the modem when the port is opened. I'm guessing Xastir is sending the config commands to quickly.

As stated before, I don't know what hardware you're running, but guessing you're using some kind of USB to serial converter, just disconnect the DTR pin of that, and I'm pretty certain it will work.

**Comment by na7q** [comment]
It's possible it's rebooting as you say. The serial converter is built into to arduino nano I'm using. But all is well, I've just set what I needed in the source. Took me a while as I'm a bit rusty. But got it. Thanks for the assistance and all the code to make this happen.

**Comment by markqvist** [comment]
Hi Mike, I investigated further, and I can confirm that it occurs when the board is rebooted by the host, which Xastir does. What then happens is that Xastir sends the config commands immediately, and they will end up being read by your Arduino's bootloader, instead of MicroAPRS.

There's a simple fix, just add a small resister, like 100 ohm, between 5V and RESET of your arduino. I also updated the README file in the repo so this info is included: https://github.com/markqvist/MicroAPRS#kiss-mode

Thanks for bringing it to my attention :)
